packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -249,9 +246,13 @@ export default function ClassifierContainer({
 }
 
 ClassifierContainer.propTypes = {
+  /** Returned from useAdminMode() in parent app */
   adminMode: PropTypes.bool,
+  /** panoptes-client/lib/auth is passed here from parent app */
```

Comment:
The dev classifier uses `panoptes-client/lib/oauth` to authenticate. I don’t know if we should focus on a specific OAuth client here. 

Also, our clients are badly named because both `auth` and `oauth` use OAuth to authenticate.

Reply:
Oohhh yes I was definitely confused about that

Reply:
Yeah, `auth` handles OAuth on `*.zooniverse.org` domains. `oauth` redirects you to a Zooniverse login page, then back to the calling app. 

They’re both quite old now, and the OAuth flows that they use are no longer recommended for use, but that’s a separate problem. 

`auth` uses https://oauth.net/2/grant-types/password/

`oauth` uses https://oauth.net/2/grant-types/implicit/